core/frontend/src/extension/Extension.ts

```diff
@@ -6,6 +6,11 @@
  * @module Extensions
  */
 
+/**
+ * @internal
+ */
+export const loggerCategory = "imodeljs-frontend.Extension";
```

Comment:
Why is this being exported from core-frontend?

Reply:
Sorry I forgot to change the "imodeljs" from the previous iteration of Extensions (v2.19).
Actually I just discovered the FrontendLoggerCategory enum... should I change the value to "core-frontend.Extensions" and add to the enum?

Reply:
That sounds right.